tailwind.config.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
     margin: ['first', 'last', 'responsive'],
     boxShadow: ['group-focus'],
     opacity: ['disabled', 'hover', 'group-hover'],
+    scale: ['group-hover'],
```

Comment:
You need to add the defaults here, such as `hover` or any other scaling hovers not using group-hover won't work.

Reply:
Hm.  I was literally testing the poster hovers though, and they didn't stop working with this addition?

Reply:
Better to assume it won't work in some situation and add it to be safe. Like if it's compiled away or removed as a default by tailwind or something. No reason not to specify it in the variants.